.github/workflows/release.yaml

```
@@ -119,7 +119,7 @@ jobs:
         run: echo LAST_COMMIT_MSG=$(git --no-pager log -1 --pretty=%B) >> $GITHUB_ENV
 
       - name: Decides if Docs should be deployed
-        if: startsWith(env.LAST_COMMIT_MSG, 'ci(changesets):') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
+        if: startsWith(env.LAST_COMMIT_MSG, 'ci:') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
```

```suggestion
        if: startsWith(env.LAST_COMMIT_MSG, 'ci: versioning packages and changesets') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
```

@nedsalk is this correct?

---

Yes, but you could also change it to check for equality instead of `startsWith` now:
```
env.LAST_COMMIT_MSG == 'ci: versioning packages and changesets'
```